### DISCUSSION TITLE
INTERNAL_SERVER_ERROR without any details in logs

### DISCUSSION THREAD

**Discussion by ziedHamdi** [vercel/community#2758] [Bug Report]
### Page to Investigate

https://test.cvlink.in/

### Steps to Reproduce

Just access the page and it will fail to load.

I complied the app locally `vercel build --prod `and tested locally `vercel run preview` everything works fine. When I deploy `vercel deploy --prebuilt`, I can't access the page.

The logs on the server don't say anything useful:
![image](https://github.com/vercel/community/assets/7271508/044a91c9-a5c1-43d2-a9e7-2b1dc7cbb3a6)

(in addition there are requests to /search that I don't know where they are coming from, but that's maybe unrelated)

In the console in chrome, I see an error caused by the call
`Br = Object.keys(yr && yr.activated_links`

I searched my local `.vercel` folder to find that line of code but it doesn't exist.

**Comment by ziedHamdi** [comment]
One thing that has changed since I last deployed my app: I was building under an ubuntu machine, and I moved to windows machine. The generated files are then different...

**Comment by mungruez** [comment]
![image](https://github.com/vercel/community/assets/16914862/e6b8fc15-3b46-4412-9d94-f1bb07b38d9c)
I made a chatGpt django app and vercel is crap , waste of time,  better they were in the garbage !!! there is no way to fix anything on vercel. No help no support and too many reasons apps work elsewhere perfectly but fail on vercel
